tests/README.md

```diff
@@ -90,3 +90,7 @@ psql service=lizmap-mapbuilder
 
 If you want to enter into the lizmap container to execute some commands,
 execute `make shell`.
+
+## Linters
```

Comment:
to avoid conflicts, you can remove this part from all other PR if you want, it will be easier